src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/ApiController.cs

```diff
@@ -132,7 +132,7 @@ public async Task<IActionResult> Create(CreateApiViewModel model)
         [Route("setup")]
         public async Task<ActionResult> Setup(SetupApiViewModel model)
         {
-            if (!_currentShellSettings.IsDefaultShell())
+            if (_currentShellSettings.Name != ShellHelper.DefaultShellName)
```

Comment:
According to your though about the changes, what's the difference if we make `IsDefaultShell()` as extension method instead of writing the comparison each time?

Reply:
Maybe https://github.com/OrchardCMS/OrchardCore/pull/12114#discussion_r933740444 for some first thoughts, then https://github.com/OrchardCMS/OrchardCore/pull/11435#issuecomment-1201890258 the most recent one I think.